testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -56,6 +56,7 @@ Feature: Smoke tests for <client>
 @skip_for_debianlike
 @skip_for_rocky9
 @skip_for_alma9
+@skip_for_sl_micro
```

Comment:
We should not be skipping them, this is a supported feature.
Instead, we need to find a way to cover this with a patch, if none available now.